vernac/metasyntax.ml

```diff
@@ -1065,16 +1088,30 @@ module SynData = struct
 
     (* Notation data for parsing *)
 
-    level         : int;
-    syntax_data   : (Id.t * (production_level, production_position) constr_entry_key_gen) list * (* typs    *)
-                    symbol list;                                                                 (* symbols *)
+    level         : level;
+    pa_syntax_data : (Id.t * (production_level, production_position) constr_entry_key_gen) list * (* typs    *)
+                     symbol list;                                                                 (* symbols *)
+    pp_syntax_data : (Id.t * (production_level, production_position) constr_entry_key_gen) list * (* typs    *)
```

Comment:
Yes, I hesitated to do so but keep it as it is.

Note that the organization of fields is not absolutely canonical. There are various more or less equivalently-justifiable ways to organize the fields, and this is one of them. Otherwise said, no big pretention in there.